.github/workflows/devtools.yml

```diff
@@ -73,3 +73,18 @@ jobs:
       with:
         name: results
         path: ${{ github.workspace }}/lighthouse/.tmp/layout-test-results
+    
+    # Run smoke tests via DevTools
```

Comment:
How to not avoid redoing all the work for building devtools?

Reply:
> How to avoid redoing all the work for building devtools?

Yeah it's not pretty. We could cache the build (or have dependent jobs)? I'm not sure of the best approach, but if the devtools CI job is normally going to take 30+ minutes we'll definitely want to split it up.